processor/transform/callouts.ts

```diff
@@ -31,8 +25,8 @@ export const wrapHeading = (node: Blockquote | Callout): Heading => {
     depth: 3,
     children: ('children' in firstChild ? firstChild.children : []) as Heading['children'],
     position: {
-      start: findFirst(firstChild)?.position.start,
-      end: findLast(firstChild)?.position.end,
+      start: firstChild.position.start,
+      end: firstChild.position.end,
```

Comment:
This was technically incorrect, and should have been using the parent position. Oops.